lib/ansible/galaxy/data/default/collection/plugins/README.md.j2

```diff
@@ -0,0 +1,31 @@
+# Collections Plugins Directory
+
+This directory can be used to ship various plugins inside an Ansible collection. Each plugin is placed in a folder that
+is named after the type of plugin it is in. It can also include the `module_utils` and `modules` directory that
+would contain module utils and modules respectively.
+
+Here is an example directory of the majority of plugins currently supported by Ansible;
+
+```
+└── plugins
```

Comment:
Maybe render smth like this?
```suggestion
{{ collection_name }}/
└── plugins
```

Reply:
I don't see much point, this is a file in the `plugins` dir already and having it render the namespace/collection is just unecessary noise IMO.